cd2h_repo_project/modules/records/jsonschemas/records/record-v0.1.0.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
   "description": "Describe information needed for deposit and record module.",
-  "id": "https://cd2hrepo.galter.northwestern.edu/schemas/records/record-v0.1.0.json",
+  "id": "galter.northwestern.edu/record-v0.1.0.json",
```

Comment:
This record doesn't exist. Is this a placeholder?

Reply:
This json is our *schema for all our records*. The rules governing the syntax/format of *this* schema file are provided by the `$schema` value. `$schema` value is not referring to our schema and the rest of the content is not a record.

`id` here just has to be different for different schemas, otherwise we would need a different url for dev, stage. and production and therefore a different schema file.